Nwjs 0.23

Profpatsch opened the pull request:
###### Motivation for this change

Remove the unmaintained old versions and update to the newest.

@matejc nixui was the only program depending on it, they seem to have changed their API

```
> result/bin/nixui 
[0803/022746.783454:WARNING:chrome_main_delegate.cc(569)] final extension: 
fs.js:953
  binding.stat(pathModule._makeLong(path));
          ^

Error: ENOENT: no such file or directory, stat '/etc/nixos/hardware-configuration.nix'
    at Object.fs.statSync (fs.js:953:11)
    at /nix/store/f2mc0v8fbrf3b23gq28j40j3d9fb184c-node-nixui-0.2.1/lib/node_modules/nixui/src/data.js:89:16
    at Array.forEach (<anonymous>)
    at HTMLElement.dbs.configurations (/nix/store/f2mc0v8fbrf3b23gq28j40j3d9fb184c-node-nixui-0.2.1/lib/node_modules/nixui/src/data.js:88:20)
```

I wouldn’t want to merge this until nixui is patched to work with `0.23`.

Profpatsch committed:
node_webkit/nwjs: 0.12 -> 0.23.6

Keep 0.12, because it is still needed by `nixui`.
Keep 0.9, because it is still needed by `zed`.

NeQuissimus commented on pkgs/tools/package-management/nixui/default.nix:
```diff
@@ -42,3 +42,4 @@ stdenv.mkDerivation rec {
     platforms = stdenv.lib.platforms.unix;
   };
 }
+/
```
Pretty sure this / should not be here :)

Profpatsch replied:
Thanks, dunno how that sneaked in. Fixed.

grahamc commented:
@GrahamcOfBorg eval

grahamc commented:
@GrahamcOfBorg eval

Profpatsch commented:
Okay, I’ve rebased this to the current master and changed it so we keep the outdated version `nixui` needs.

Meanwhile, `nwjs` already progressed to `0.29`, but a version a little non-ancient can’t hurt. :)

Profpatsch commented:
@GrahamcOfBorg eval

Profpatsch commented:
@GrahamcOfBorg eval

grahamc commented:
Just a note @Profpatsch, after you push to the branch an eval is automatically scheduled, so no need to call `eval` manually :)
